core-java-modules/core-java-lang-oop-modifiers/src/test/java/com/baeldung/publicmodifier/PublicAccessModifierUnitTest.java

```diff
@@ -64,9 +65,9 @@ public void whenParsingStringOne_parseIntReturns1() {
     @Test
     public void whenConnectingToH2_connectionInstanceIsReturned() throws SQLException {
 
-        final String url = "jdbc:h2:~/test";
+        final String url = "jdbc:h2:mem:test";
```

Comment:
let's use the filesystem db, not the in-memory one.

Reply:
It's already done in the new PR. Closing this PR